omniauth-polaris.gemspec

```diff
@@ -1,29 +1,30 @@
-# -*- encoding: utf-8 -*-
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'omniauth-polaris/version'
-Gem::Specification.new do |gem|
-  gem.name = "omniauth-polaris"
-  gem.version = OmniAuth::Polaris::VERSION
-  gem.authors = ["Steven Anderson", "Ben Barber"]
-  gem.email = ["sanderson@bpl.org", "bbarber@bpl.org"]
-  gem.description = %q{A Polaris API strategy for OmniAuth.}
-  gem.summary = %q{A Polaris API strategy for OmniAuth.}
-  gem.homepage = "https://github.com/boston-library/omniauth-polaris"
+# frozen_string_literal: true
+
+$:.push File.expand_path('lib', __dir__)
+require 'omniauth/polaris/version'
 
-  gem.required_ruby_version = '~> 2.4'
+Gem::Specification.new do |gem|
+  gem.name = 'omniauth-polaris'
+  gem.version = OmniAuth::Polaris::VERSION.dup
+  gem.authors = ['Steven Anderson', 'Ben Barber']
+  gem.email = ['sanderson@bpl.org', 'bbarber@bpl.org']
```

Comment:
Let's change Steven's email to `digital@bpl.org`

Reply:
Do you want me to add you in there as well?